flink-runtime/src/test/java/org/apache/flink/runtime/rest/handler/legacy/backpressure/BackPressureStatsTrackerTest.java

```diff
@@ -80,11 +73,11 @@ public void execute(Runnable runnable) {
 
 		ExecutionVertex[] taskVertices = new ExecutionVertex[4];
 
-		ExecutionJobVertex jobVertex = mock(ExecutionJobVertex.class);
-		when(jobVertex.getJobId()).thenReturn(new JobID());
```

Comment:
these changes are all unnecessary.

Reply:
I think these changes are the result of the automatic import optimization.